platform.md

```diff
@@ -1123,8 +1137,9 @@ Where:
 Where:
 - `app` MUST contain one entry per app slice layer where
   - `sha` MUST contain the digest of the uncompressed layer
-- `config.sha` MUST the digest of the uncompressed layer containing launcher config
-- `launcher.sha` MUST the digest of the uncompressed layer containing the launcher binary
+- `bom.sha` MUST contain the digest of the uncompressed layer containing buildpack-provided standardized Bill of Materials
```

Comment:
Are we keeping this in lifecycle metadata or moving it to its own key?

Reply:
Per sub sync discussion, we plan to leave the sha in lifecycle metadata for now. In the future, we may duplicate this information in a new label.